src/pkg/packaging/snaps/dotnet-sdk/dotnet-runtime

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
```

Comment:
Any reasonable way to share this script?

Reply:
Probably something we can do while re-architecting for build automation.